profile.css

```diff
@@ -1 +1,69 @@
-// style goes here
+/* style goes here */
+:root{
+    --Gray-3: #828282;
+    --Gray-4:#BDBDBD;
+    --Gray-5: #E0E0E0;
+    --Gray-6: #F2F2F2;
+    --crypto-eclips: #0C445B;
+}
+*{
+  box-sizing: border-box;
+}
+.container{
+border-radius: 12px;
+border: 2px solid var(--Gray-4);
+width: 80%;
+margin: 0 auto;
+font-family: 'Poppins', sans-serif;
+padding:1.5em 3.125em 3.125em;
+}
+.heading{
+  margin-bottom: 1.5em;
+}
+.heading h1{
+  font-size: 1.5rem;
+  font-weight: 700;
+  line-height: 32px;
+  margin-bottom: .4em;
+}
+.heading p{
+  font-size: 1.125rem;
+  font-weight: 500;
+  line-height: 24px;
+  margin-top: 0;
+}
+input{
+  width: 100%;
+  padding: 12px;
+  margin: 0.8em 0 2.5em 0;
+  display: inline-block;
+  border-radius: 8px;
+  border: 1px solid var(--Gray-5);
+  opacity: 0.8;
+  background: var(--Gray-6);
+  box-shadow: 0px 1px 4px 0px rgba(255, 255, 255, 0.69);
```

Comment:
be consistent with using css variables, make it cleaner.. good work

Reply:
Noted sir.